### ISSUE TITLE
Webhook page doesn't consider map path

### ISSUE THREAD

**Issue by jwu-au** [Xabaril/AspNetCore.Diagnostics.HealthChecks#141]
Hi,

If boot up the UI via
```C#
const string mapPath = "/healthchecks";
app.Map(mapPath, builder => { builder.UseHealthChecksUI(); });
```

Then when going to webhook web page in the UI, it hits the webhook json url results in 404.
It should consider the map path defined above.

Thanks,
Jing

**Comment by unaizorrilla** [comment]
Ping @CarlosLanderas

**Comment by CarlosLanderas** [comment]
Should be fixed in version 2.2.30
#209

Webhooks path was not using relative path, and now you can configure all paths.
I close the issue. 

Feel free to reopen it if this is not working for you.
Thanks!
